pandas/core/series.py

```diff
@@ -3098,11 +3098,16 @@ def nlargest(self, n=5, keep='first'):
             When there are duplicate values that cannot all fit in a
             Series of `n` elements:
 
-            - ``first`` : take the first occurrences based on the index order
-            - ``last`` : take the last occurrences based on the index order
+            - ``first`` : return the first `n` occurrences in order
+                of appearance.
+            - ``last`` : return the last `n` occurrences in reverse
+                order of appearance.
             - ``all`` : keep all occurrences. This can result in a Series of
                 size larger than `n`.
 
+            Regardless of `keep`, the result will be sorted
```

Comment:
not sure what this means here. This is very misleading. This is dependent on the ordering of the index itself. I am not sure if this is an actual bug or not. cc @chris-b1

Reply:
@chris-b1 @jreback shall I just remove this? I wanted to indicate that`order of appearance` meant `index order`, but this only seems to add to the confusion